Caishen.podspec

```diff
@@ -29,5 +29,5 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.source_files = 'Pod/Classes/**/*'
-  s.resources = 'Pod/Assets/*'
+  s.resources = ['Pod/Assets/*', 'Pod/Resources/*']
```

Comment:
The difference is more or less the kind of content. `Assets` for image resources and the like, whereas `Resources` in this case are mostly .strings files for the localization.